src/diet.ts

```diff
@@ -316,6 +316,12 @@ function menu(): MenuItem<Note>[] {
    *  > js Item.all().filter((item) => item.fullness > 0 && item.name.indexOf("lasagna") > 0 && getIngredients(item)["savory dry noodles"]).join(", ")
    */
   const lasagnas = $items`fishy fish lasagna, gnat lasagna, long pork lasagna`;
+
+  /* 
+   * standardSpleenItem indicates a spleen item of size 4 with an adventure yield of 5-10. Taken from the wiki. They are all functionally equivalent.
```

Comment:
wow what an incredible comment whoever wrote this must be a really strong individual with a great spirit and a happy heart, who has never once in their life experienced depression. incredible. great stuff.

Reply:
now that i've removed trailing spaces this is even more beautiful